proto/api/v1/audit.proto

```diff
@@ -35,6 +35,38 @@ message Audit {
             string endpoint             = 1;
             string method               = 2;
             google.protobuf.Any payload = 3;
+            // Source holds information about the request's source.
+            message Source {
+                // Holds the value of the "X-Forwarded-For" header associated with the request, if it was present.
+                string x_forwarded_for     = 2;
+                // Holds the value of the "Remote-Addr" header associated with the request, if it was present.
+                string remote_addr         = 3;
+                // Holds the value of the HTTP request's address. This will always be present, but may not preserve
+                // the _actual_ client IP. See source_ip for a more detailed documentation.
+                string request_addr = 4;
+            }
+            Source source                  = 5;
```

Comment:
I thought about naming this `SourceHeaders` instead. Then it's also a bit less awkward to have `source_ip` separate from `source`.